composer.json

```diff
@@ -18,9 +18,19 @@
   "homepage": "https://github.com/atk4/data",
   "license": "MIT",
   "authors": [
-    {"name": "Romans Malinovskis", "email": "romans@agiletoolkit.org", "homepage": "https://nearly.guru/" },
-    {"name": "Imants Horsts", "homepage": "https://darkside.lv/"},
-    {"name": "Michael Voříšek", "homepage": "https://mvorisek.cz/"}
+    {
```

Comment:
By hand, or is there any composer command to prettify/normalize composer.json?

Reply:
I did it by hand :)
Thing is that json will be prettified anyway when making release, but to keep changes small when releasing it's good to have already prettified json.

Reply:
I put it there previously "minified" to lower the total real estate... And that is what I usually see also in other repos, but ok with me if you do not want to change the release processes...